pom.xml

```diff
@@ -24,12 +24,19 @@
   </licenses>
 
   <scm>
-    <connection>scm:git:git@github.com:cloudbees/super-simple-disk-usage-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:cloudbees/super-simple-disk-usage-plugin.git</developerConnection>
-    <url>http://github.com/cloudbees/super-simple-disk-usage-plugin</url>
+    <connection>scm:git:git@github.com:jenkinsci/cloudbees-disk-usage-simple.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/cloudbees-disk-usage-simple.git</developerConnection>
+    <url>https://github.com/jenkinsci/cloudbees-disk-usage-simple</url>
     <tag>HEAD</tag>
   </scm>
 
+  <distributionManagement>
+    <repository>
+      <id>maven.jenkins-ci.org</id>
+      <url>http://maven.jenkins-ci.org/content/repositories/releases/</url>
```

Comment:
I think this might need the port number (8081)?

And are we sure this needs to be added? This is already defined in `org.jenkins-ci:jenkins` (parent of `org.jenkins-ci.plugins:plugin`).